setup.py

```diff
@@ -90,6 +90,10 @@
     "pytz",
     "characteristic >= 14.1.0",
     "Twisted == 15.1.0",
+    # TLS support libraries for Twisted:
+    "service_identity == 14.0.0",
```

Comment:
In general they ought to be kept up to date, yes, although we don't actually rely on service_identity or idna for security.